website/src/PageWithSidebar.js

```diff
@@ -62,15 +62,14 @@ class PageWithSidebar extends Component {
                 state: route,
                 dispatch,
               });
-              const linkName = router.getScreenConfig(childNavigation, 'linkName');
-              const icon = router.getScreenConfig(childNavigation, 'icon');
+              const options = router.getScreenOptions(childNavigation, {});
               const isActive = state.index === i;
               return (
                 <span key={i}>
                   <LinkableLi
                     to={route.routeName}
-                    className={"pt-menu-header "+icon+' '+(isActive?'active':'')}>
-                    <h6>{linkName}</h6>
+                    className={"pt-menu-header "+options.icon+' '+(isActive?'active':'')}>
+                    <h6>{options.linkName}</h6>
```

Comment:
We can now make it more specific (`Options` are separate for each navigator and not shared). That means we can tell Flow the specific type we are expecting.